run/filesystem/index.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const express = require('express');
+const rateLimit = require('express-rate-limit');
+const fs = require('fs');
+const serveIndex = require('serve-index');
+
+const app = express();
+const limit = rateLimit({
+  windowMs: 15 * 60 * 1000,
+  max: 100,
+  message: 'Rate limit exceeded',
+  headers: true,
+});
+const mntDir = process.env.MNT_DIR || '/mnt/nfs/filestore';
+const filePrefix = process.env.FILENAME || 'test';
+const port = parseInt(process.env.PORT) || 8080;
+
+app.use(limit);
+app.use('/filesystem', express.static(mntDir));
+app.use('/filesystem', serveIndex(mntDir));
```

Comment:
Do we need to prefix with filesystem? The other languages do not

Reply:
updated to use the same paths as python sample